src/olympia/devhub/templates/devhub/new-landing/components/footer.html

```diff
@@ -7,8 +7,9 @@ <h3 class="DevHub-Footer-sections-header">{{ _('Resources') }}</h3>
         <h4>{{ _('Documentation') }}</h4>
         <ul>
           <li><a href="https://developer.mozilla.org/Add-ons">{{ _('Add-ons') }}</a></li>
-          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('WebExtensions') }}</a></li>
+          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('Extensions') }}</a></li>
           <li><a href="https://developer.mozilla.org/Add-ons/Themes/Background">{{ _('Themes') }}</a></li>
```

Comment:
> That's exactly why I'm saying we should hard-code en-US.

Ah ok my bad - I took "language slug" to mean current language since there was no mention of the default english locale. Seems a shame to not get the benefit of translations if they exist but I can see the case for forcing en-US if it's difficult to guarantee translations would be up to date. Presumably MDN does not inform the user if the translation is older than the page it's a translation of?